src/app/profile/flag/flag.component.ts

```diff
@@ -1,27 +1,39 @@
 import { Component, Input, ChangeDetectionStrategy } from '@angular/core';
 import { Country } from '../country';
 
+declare var cordova: any;
+
 /**
  * Component for displaying a user's flag
  */
 @Component({
   selector: 'flag',
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <img *ngIf="countryCode"
+    <img *ngIf="countryCode && !isCordova"
       [title]="country(countryCode)"
       srcset="
         /assets/img/flags/16/{{ countryCode.toLowerCase() }}.png 1x,
         /assets/img/flags/32/{{ countryCode.toLowerCase() }}.png 2x
       "
       src="/assets/img/flags/16/{{ countryCode.toLowerCase() }}.png"
     >
+    <img *ngIf="countryCode && isCordova"
+      [title]="country(countryCode)"
+      srcset="
+        assets/img/flags/16/{{ countryCode.toLowerCase() }}.png 1x,
+        assets/img/flags/32/{{ countryCode.toLowerCase() }}.png 2x
+      "
+      src="assets/img/flags/16/{{ countryCode.toLowerCase() }}.png"
+    >
   `
 })
 export class FlagComponent {
 
   @Input() countryCode: string;
 
+  isCordova = typeof cordova !== 'undefined';
```

Comment:
Please add:
```ts
export const IS_CORDOVA = typeof cordova !== 'undefined';
```
to `app.module.ts` or something else and import the constant from there in every component where it's needed.